README.md

```diff
@@ -54,7 +54,7 @@ Finally, you can create an instance of your custom resource (CR).
 Once an instance exists, your operator will take over managing it.
 
 ```bash
-$ oc create -f deploy/cr.yaml
+$ oc create -f examples/cr.yaml
```

Comment:
I don't think this is needed at all anymore. @benjaminapetersen can you confirm?

Reply:
No we dont... the instance is created automatically